### ISSUE TITLE
Add 2021 Swag shirt and stickers to history section.

### ISSUE THREAD

**Issue by zo0o0ot** [hacktoberfesthowto/howto-blog#21]
Best example I can find is in this tweet:
https://twitter.com/thesnjvsharma/status/1491325642276995074
![hf2021-shirt](https://user-images.githubusercontent.com/876146/189814761-f227194c-8594-452e-8e14-e99513f3c250.jpg)
![hf-2021-stickers](https://user-images.githubusercontent.com/876146/189814767-4dd24ccb-6df5-4883-a5c4-761dfd8b8b36.jpg)

The images should be posted to https://github.com/hacktoberfesthowto/howto-blog/tree/main/static/images

References to images should go in https://github.com/hacktoberfesthowto/howto-blog/blob/main/content/history.md

**Comment by yashkhandelwal2203** [comment]
hey, can you please tell what exactly i have to do in this contribution. i am new to open source contribution . so it would be great if you could help me.

**Comment by zo0o0ot** [comment]
This repo is the source code for a static site, written using [Hugo](https://gohugo.io/) and Markdown.
Instead of installing the framework, I set up the repo to run in [Gitpod](https://www.gitpod.io/).

What you'll want to do:

Option 1: Change inside of Gitpod and preview the changes

- Make a fork of this repo to your local user.
- Sign up for a free Gitpod account.  You can use your GitHub user to sign in.
- Open your fork in Gitpod (it may be useful to install the Gitpod extension for your browser for this.
-- The URL to open will look something like https://gitpod.io/#https://github.com/yourusername/howto-blog
- Download the two images that are attached to this GitHub issue.
- Copy them to the howto-blog/tree/main/static/images directory.
- Open the howto-blog/blob/main/content/history.md file.
- Look near the bottom of the file. 
- Underneath `{{< figure src="/images/2021Logo.webp" width="1000" >}}`, you will want to add entries for the two new files that you added.  It will look something like this:

```
{{< figure src="/images/image1.jpg" width="1000" >}}

{{< figure src="/images/image2.jpg" width="1000" >}}
```

You should be able to see a preview of this inside of Gitpod.


Option 2: Make the change directly in GitHub.
- Make a fork of this repo to your local user.
- Download the two images that are attached to this GitHub issue.
- Upload the files to your fork, in the appropriate folder, like this: 
![image](https://user-images.githubusercontent.com/876146/190046400-81d4b0cd-4f3e-45e1-9bdf-820a1ea26728.png)
- Go to the howto-blog/blob/main/content/history.md file, and Click the pen to edit the file.
![image](https://user-images.githubusercontent.com/876146/190047149-02d4f976-589d-44d2-a489-05b47ed7c290.png)
- Look near the bottom of the file. 
- Underneath `{{< figure src="/images/2021Logo.webp" width="1000" >}}`, you will want to add entries for the two new files that you added.  It will look something like this:

```
{{< figure src="/images/image1.jpg" width="1000" >}}

{{< figure src="/images/image2.jpg" width="1000" >}}
```







Once you have finished, open a pull request, and we'll go from there!

**Comment by yashkhandelwal2203** [comment]
ok once done i will open a pull request. Thanks for the guidance and support .

**Comment by zo0o0ot** [comment]
Everything looks great!

Closed with #25 .
I updated some filenames with #27
